lib/event-source.js

```diff
@@ -0,0 +1,79 @@
+var mongojs = require('mongojs');
+var inflect = require('i')();
+var _ = require('lodash');
+var EventSource = function() {
+};
```

Comment:
the spec defines 2 ways of getting change events, streaming and non streaming :
https://github.com/agco/json-api-search-profile/blob/master/public/change-events-profile.md#stream-changes

streaming is implemented here, but should therefore be suffixed with /stream